docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit/add-certificate-ca.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Add CA-signed certificates to YugabyteDB Anywhere
+headerTitle: Add certificates
+linkTitle: Add certificates
+description: Add CA-signed certificates to YugabyteDB Anywhere.
+headcontent: Use your own certificates for encryption in transit
+menu:
+  preview_yugabyte-platform:
+    parent: enable-encryption-in-transit
+    identifier: add-certificate-2-ca
+    weight: 20
+type: docs
+---
+
+{{<tabs>}}
+{{<tabitem href="../add-certificate-self/" text="Self-Signed" >}}
+{{<tabitem href="../add-certificate-ca/" text="CA-Signed" active="true" >}}
+{{<tabitem href="../add-certificate-hashicorp/" text="Hashicorp Vault" >}}
+{{<tabitem href="../add-certificate-kubernetes/" text="Kubernetes" >}}
+{{</tabs>}}
+
+For universes created with an on-premises provider, instead of using self-signed certificates, you can use third-party certificates from external certificate authorities (CA). The third-party CA root certificate must be configured in YugabyteDB Anywhere. You also have to copy the custom CA root certificate, node certificate, and node key to the appropriate on-premises provider nodes.
+
+## Prerequisites
+
+The certificates must adhere to the following criteria:
+
+- Be stored in a `.crt` file, with both the certificate and the private key being in the PEM format.
+
+  If your certificates and keys are stored in the PKCS12 format, you can [convert them to the PEM format](#convert-certificates-and-keys-from-pkcs12-to-pem-format).
+
+- Contain IP addresses of the database nodes or DNS names as the Subject Alternative Names (wildcards are acceptable).
+
+## Add CA-signed certificates
+
+The following procedure describes how to install certificates on the database nodes. You have to repeat these steps for every database node that is to be used in the creation of a universe.
+
+### Obtain certificates and keys
+
+Obtain the keys and the custom CA-signed certificates for each of the on-premise nodes for which you are configuring node-to-node TLS. In addition, obtain the keys and the custom signed certificates for client access for configuring client-to-node TLS.
+
+### Copy the certificates to each node
+
+For each on-premises provider node, copy the custom CA root certificate, node certificate, and node key to that node's file system.
+
+If you are enabling client-to-node TLS, make sure to copy the client certificate and client key to each of the nodes.
```

Comment:
```suggestion
If you are enabling client-to-node TLS, make sure to copy the client-facing server certificate and client-facing server key to each of the nodes.
```

The language we use in our own UI is a bit confusing. A client certificate is typically used by a client. In this case, we are talking about server certificates used for the YSQL/YCQL client interfaces, so I've tried to clarify the language. However, it is not consistent with our own UI, so please feel free to ignore this suggestion.